### ISSUE TITLE
Add releases/tags

### ISSUE THREAD

**Issue by kfoxb** [dolthub/jsplit#2]
Would it be possible to create some releases and/or add some tags? It would help communicate where the project is currently at, what commits are adding, and be easier to specify versions when the software is used.

**Comment by timsehn** [comment]
This is kind of a side hustle for us. We may not create a release.

**Comment by bheni** [comment]
Yeah this is just a tool I created to help bounty participants who were working on hospital pricing data whose format was out of control.  I'm happy to fix bugs as they come up, but this isn't a very active project.
